bbox-feature-server/src/datasource/postgis.rs

```diff
@@ -143,17 +157,21 @@ pub struct PgCollectionSource {
     geometry_column: String,
     /// Primary key column, None if multi column key.
     pk_column: Option<String>,
+    temporal_column: Option<String>,
+    other_columns: HashMap<String, u8>,
```

Comment:
What does the `u8` value mean? Shouldn't it be an array of strings for allowed values?

Reply:
I've pushed another commit to use HashSet instead of HashMap